VCServices/VCServices/IssuanceService.swift

```diff
@@ -82,6 +82,12 @@ public class IssuanceService {
         }
     }
     
+    public func send(completionResponse response: IssuanceCompletionResponse, to url: String) -> Promise<String?> {
```

Comment:
nit, can we rename this to something like `sendIssuanceCallback`? Seems ambiguous together with the "main" `send`

Reply:
yes sounds good